CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 otj-metrics
 ===========
 
+4.0.0
+-----
+* Uses DropWizard 4.0.2
+
+This is an incompatible binary/source change, which will require users to import new versions, and a new 
+module (metrics-jmx)
+
+In addition, a fork of metrics-spring will probably be needed
```

Comment:
We should reach out to metrics-spring maintainer and suggest that he either:
 a) quickly update his project
 b) add us as maintainers and allow us to update his project for him
 c) mark his project as abandoned in favor of a fork that we build

Otherwise, we can either give up on the library, or we can build our own fork.

Reply:
@adamzr I have a fork and a PR put up. Take a look at https://github.com/ryantenney/metrics-spring the original github. Pretty much dead eh? Look at the PRs that have waited for years.